http/swagger.yml

```diff
@@ -6781,6 +6782,27 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Expression"
+    DictExpression:
+      description: Used to create and directly specify the elements of a dictionary
+      type: object
+      properties:
+        type:
+          $ref: "#/components/schemas/NodeType"
+        elements:
+          description: Elements of the dictionary
+          type: array
+          items:
+            $ref: "#/components/schemas/Expression"
```

Comment:
Should this be a list of DictItem instead? Or should DictItem be added to the set of Expressions above?

Reply:
No you're right, typo on my part. Thanks.